proposals/Modules.md

```diff
@@ -0,0 +1,65 @@
+## Modules
+
+Modules will provide the basic organization building block in Serulian.
+
+### Basic Definition
+
+A module is defined as a single Serulian file (ending in `.seru`) that **exports** some set of types
+or functions to be accessible to other modules.
+
+### Using modules
+
+Modules can be **imported** using the `from` or `import` keywords, which support import
+from two different sources: local file system and Git.
+
+Syntax Examples:
+
+```
+// Imports the file "myfavoritemodule.seru" directly, adding its exported members into the local context
+import myfavoritemodule
```

Comment:
Do we want to require semi-colons or try to be more Go-like in using newlines?  (personally, I prefer the Go-way)